core-build/web-library-build/src/index.ts

```diff
@@ -52,8 +52,8 @@ tslint.mergeConfig({
 });
 
 // Define default task groups.
-export const compileTsTasks: IExecutable = parallel(typescript, text, apiExtractor);
```

Comment:
>apiExtractor [](start = 70, length = 12)

FYI all the other rigs were already updated to run api-extractor after tsc; I think nobody is using web-library-build any more?